### DISCUSSION TITLE
Support throwing validation errors

### DISCUSSION THREAD

**Discussion by kesar** [next-safe-action/next-safe-action#171] [Ideas]
Is there any way to throw exceptions for validation errors?

```
handleServerErrorLog(e) {
    console.error(e.message)
    throw e
  },
  handleReturnedServerError(e) {
    throw e
  },
```

this doesnt seem to catch those. I am not using useAction and getting return into useMutation of react query. Unfortunately this goes to onSuccess() there instead

**Comment by TheEdoRan** [comment]
No way currently to do that. Or you could pass `handleValidationErrorsShape` to every `schema` method you use, but it's not that great. I think a `throwValidationErrors` option at the client level would be a good solution for this. Thanks for the suggestion.

**Comment by TheEdoRan** [comment]
Added support for this feature in [v7.1.0](https://github.com/TheEdoRan/next-safe-action/releases/tag/v7.1.0)
